forte/data/data_store.py

```diff
@@ -428,6 +459,23 @@ def _new_group(
 
         return entry
 
+    def _new_generics(self, type_name: str, tid: Optional[int] = None):
+        r"""This function generates a new generics with default fields. All
+        default fields are filled with None.
+        Called by add_generics_raw() to create a new generics with
+        ``type_name``.
+
+        Args:
+            type_name: The fully qualified type name of the new entry.
+            tid: ``tid`` of the generics entry.
+
+        Returns:
+            A list representing a new generics type entry data.
+        """
+        tid: int = self._new_tid() if tid is None else tid
+        entry = [tid, type_name]
```

Comment:
``tid`` and ``type_name`` should always be the third and the fourth fields (refer to https://github.com/asyml/forte/blob/961c5df923ebfe2e8d907093266d3f5ecfa4aa57/forte/data/data_store.py#L85). In `Generics` maybe we can put some placeholder like `None` in the first and second fields if there is no better ideas.

Reply:
thanks, we can use the constants to get these positions